sys/include/net/gnrc/pktbuf.h

```diff
@@ -9,7 +9,7 @@
 
 /**
  * @defgroup    net_gnrc_pktbuf   Packet buffer
- * @ingroup     net
+ * @ingroup     net_gnrc
  * @brief       A global network packet buffer.
```

Comment:
Though there are helper functions in here with regards to my definition in https://github.com/RIOT-OS/RIOT/pull/3676#discussion_r37591441 I'm still not sure why I should explicitly state here, that there are some: The number of helper functions is way lower than actual API functions (1 vs. 7, 4 vs. 7 if you count the ones only available with certain flags), so I would find it weird to mention them here.